### DISCUSSION TITLE
Trying to create button in SULU admin panel to trigger new site static build

### DISCUSSION THREAD

**Discussion by edofurlan** [sulu/sulu#6990] [Q&A] [answered]
Hi, I'm trying to add a custom button to the admin panel and connect it to a deployment pipeline on a repository using cURL, but I did not find any mention of a way to add a button in the sulu documentation. Is there a way to do that?

**Comment by alexander-schranz** [comment, marked as answer]
I personally would not trigger builds via the Admin. As the Admin should be for content managers and not trigger developer specific tasks. Devs should normally use the CLI and do there the things they need todo. And normally the website build should be tested before being deployed.

Nevertheless example to add a toolbar action can be found here:

 - https://github.com/sulu/sulu-demo/pull/81
 - https://github.com/sulu/sulu-demo/pull/82

The toolbar action are flexible enough to call whatever you want to call inside of them.
